pinot-server/src/main/java/com/linkedin/pinot/server/starter/helix/SegmentFetcherAndLoader.java

```diff
@@ -71,15 +74,18 @@ public SegmentFetcherAndLoader(DataManager dataManager, ZkHelixPropertyStore<ZNR
     } catch (Exception e) {
       // Keep the default value
     }
-    _segmentLoadMinRetryDelayMs = minRetryDelayMillis;
+    if (minRetryDelayMillis != Integer.parseInt(CommonConstants.Server.DEFAULT_SEGMENT_LOAD_MIN_RETRY_DELAY_MILLIS)) {
```

Comment:
Same here after changing the data type of CommonConstants.Server.DEFAULT_SEGMENT_LOAD_MIN_RETRY_DELAY_MILLIS to long